src/tokenserver/support.rs

```diff
@@ -119,27 +119,45 @@ impl Clone for Box<dyn VerifyToken> {
 #[derive(Clone)]
 /// An adapter to the PyFxA Python library.
 pub struct OAuthVerifier {
-    pub fxa_oauth_server_url: Option<String>,
+    inner: Py<PyAny>,
 }
 
 impl OAuthVerifier {
     const FILENAME: &'static str = "verify.py";
+
+    pub fn new(fxa_oauth_server_url: Option<String>) -> Result<Self, Error> {
+        let inner: Py<PyAny> = Python::with_gil::<_, Result<Py<PyAny>, PyErr>>(|py| {
+            let code = include_str!("verify.py");
+            let module = PyModule::from_code(py, code, Self::FILENAME, Self::FILENAME)?;
+            let kwargs = fxa_oauth_server_url
+                .clone()
```

Comment:
nit: this clone (or alternatively the other in `ServerState::from_settings`) isn't needed.